src/util.ts

```diff
@@ -150,12 +156,7 @@ export function toolPathInEnv(name: string): string | undefined {
     // (the concept of kit for cmake extension)
 
     return envPathSplit.find(p => {
-        let fullPath: string = path.join(p, path.basename(name));
```

Comment:
If we would remove quotes before we call checkFileExistsSync here, we are not covered when we call checkFileExistsSync from somewhere else where we don't verify the same thing.
Best to remove these quotes right inside checkFileExistsSync, before the stat API call.